cmake/GetVersion.cmake

```diff
@@ -1,95 +1,95 @@
 # This CMake function sets the following variables:
 #
-# ${PREFIX}_VERSION_STR       v0.6.3
-# ${PREFIX}_VERSION           0.6.3
-# ${PREFIX}_MAJOR_VERSION     0
-# ${PREFIX}_MINOR_VERSION     6
-# ${PREFIX}_PATCH_VERSION     3
-# ${PREFIX}_RELEASE           1.ci_tests_release.20180823git1cd25c2f
-# ${PREFIX}_GIT_REV           1cd25c2f
-# ${PREFIX}_GIT_BRANCH        ci-tests
+# ${PREFIX}_VERSION_STR	   v0.6.3
+# ${PREFIX}_VERSION		   0.6.3
+# ${PREFIX}_MAJOR_VERSION	 0
+# ${PREFIX}_MINOR_VERSION	 6
+# ${PREFIX}_PATCH_VERSION	 3
+# ${PREFIX}_RELEASE		   1.ci_tests_release.20180823git1cd25c2f
+# ${PREFIX}_GIT_REV		   1cd25c2f
+# ${PREFIX}_GIT_BRANCH		ci-tests
```

Comment:
I think the old version is prettier. This change seems bad.

Reply:
Agreed. I will fix it. My editor messed it up again :(